pages/linux/mkfs.f2fs.md

```diff
@@ -0,0 +1,12 @@
+# mkfs.f2fs
+
+> Creates a F2FS filesystem inside a partition.
+> More information: <https://manned.org/mkfs.f2fs>.
+
+- Create an F2FS filesystem inside partition 1 on device b (`sdb1`):
```

Comment:
```suggestion
- Create an F2FS filesystem on a block device (`/dev/sdb1`):
```

I think better to keep it generic and call it *block device*. NVMe drives have very different naming like `nvme0`. The partition can also be made be a logical volume like `/dev/vg0/lv0`.

Reply:
I copied the wording from mkfs.ext4: https://github.com/tldr-pages/tldr/blob/main/pages/linux/mkfs.ext4.md
Now that I look at them. Almost all of the mkfs. commands have the same wording.

Reply:
In that case, I guess okay to leave it as it is. Maybe later we can update all the `mkfs.*` pages and refer to partitions as block devices.